hw7.py

```diff
@@ -0,0 +1,26 @@
+phonebook = {}
+while True:
+    user_input = input('Enter command: ')   # format command name phone
+    split_input = user_input.split()
+    command = split_input[0]
+    name = split_input[1]
+    phone = split_input[2]
+    if phonebook.get(name) is None:
+        if command == 'add':
+            phonebook[name] = phone

```

Comment:
You should check if key already exists before adding a value here

Reply:
what's the difference if I am checking if key doesn't exist, then you can add it?

Reply:
I agree